adapters/orbidder/orbidder_test.go

```diff
@@ -32,3 +50,116 @@ func TestJsonSamples(t *testing.T) {
 
 	adapterstest.RunJSONBidderTest(t, "orbiddertest", bidder)
 }
+
+var testCasesCurrency = map[string]struct {
+	imp         openrtb2.Imp
+	setMock     func(m *mock.Mock)
+	expectedImp openrtb2.Imp
+}{
+	"EUR: no bidfloor, no currency": {
+		imp: openrtb2.Imp{
+			BidFloor:    0,
+			BidFloorCur: "",
+		},
+		setMock: func(m *mock.Mock) {},
+		expectedImp: openrtb2.Imp{
+			BidFloor:    0,
+			BidFloorCur: "EUR",
+		},
+	},
+	"EUR: bidfloor, no currency": {
+		imp: openrtb2.Imp{
+			BidFloor:    1,
+			BidFloorCur: "",
+		},
+		setMock: func(m *mock.Mock) {},
+		expectedImp: openrtb2.Imp{
+			BidFloor:    1,
+			BidFloorCur: "EUR",
+		},
+	},
+	"EUR: bidfloor and currency": {
+		imp: openrtb2.Imp{
+			BidFloor:    1,
+			BidFloorCur: "EUR",
+		},
+		setMock: func(m *mock.Mock) {},
+		expectedImp: openrtb2.Imp{
+			BidFloor:    1,
+			BidFloorCur: "EUR",
+		},
+	},
+	"USD: bidfloor with currency": {
```

Comment:
Can you please add another similar test case where the mock converter returns an error to make sure the error from the converter is properly propagated?

Reply:
Done.